db_startup.py

```diff
@@ -0,0 +1,56 @@
+from db_classes import TaskType, Field, ValueType, User
+from db_helper import connectToDatabase, disconnectFromDatabase
+
+
+
+# NOTE: currently not in use
+# # function that creates the admin user and saves it to the database 
+# def createAdminUser():
+#     connection = connectToDatabase()
+
+#     admin_user = User(username="admin", display_name="admin", hash_password="password", hash_salt="password", workspaces=[], tasks=[], owned_filtered_views=[], shared_filtered_views=[])
+#     admin_user.save()
+
+#     disconnectFromDatabase(connection)
+
+
+# function that creates the default task type and saves it to the database
+def createDefaultTaskType():
+
+    # create a Tag Value Type
+    tag_value_type = ValueType(name="Tag", allowed_values=[])
```

Comment:
Not something to worry about at this point but I wonder if we should provide a single allowed value in the future?